src/game_api/level_api.cpp

```diff
@@ -462,6 +462,7 @@ std::array g_community_tile_codes{
             };
             g_attachee_requiring_entities.push_back({{{x, y - 1}}, do_spawn});
         }},
+    CommunityTileCode{"boombox", "ENT_TYPE_ITEM_BOOMBOX"},
```

Comment:
Is this in the community doc?

Reply:
yeah, we were comparing the ML and OL tile codes and this one was missing, it's in the first wave in the doc